lbz/request.py

```diff
@@ -64,7 +64,9 @@ def raw_body(self):
 
     @property
     def json_body(self):
-        content_type = self.headers.get("Content-Type", self.headers.get("content-type"))
+        content_type = self.headers.get(
+            "Content-Type", self.headers.get("content-type")
```

Comment:
We can add black config file and set 90 lines as default width becuse 80 is too tight IMO

Reply:
I consider even 99